lib/preprocess.js

```diff
@@ -143,13 +143,18 @@ export const fetchMeasurements = async cid => {
       for await (const buf of entry.content()) {
         bufs.push(buf)
       }
-      const json = Buffer.concat(bufs).toString()
-      return JSON.parse(json)
+      return parseMeasurements(Buffer.concat(bufs).toString())
```

Comment:
Is it time to introduce `bl` module as discussed in https://github.com/filecoin-station/spark-evaluate/pull/70#discussion_r1408913253?

Reply:
What are you basing this on? I think it's independent of this PR. I don't know if it would help, someone needs to measure and try